tests/manager/manager-ctor-write_concern-005.phpt

```diff
@@ -0,0 +1,74 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): write concern options (64-bit wtimeoutms)
+--SKIPIF--
+<?php if (4 !== PHP_INT_SIZE) { die('skip Only for 32-bit platform'); } ?>
+--FILE--
+<?php
+
+$tests = [
+    ['mongodb://127.0.0.1/?wtimeoutms=2147483648', []],
```

Comment:
[`wtimeout`](https://docs.mongodb.com/manual/reference/write-concern/#wtimeout) is only applicable when `w` is greater than 1. I realize this isn't checked during URI parsing, but we can probably remove this since 2147483648 (i.e. `INT32_MAX + 1`) is already tested in the next line. Likewise for the fifth entry in this array.